_layouts/article.html

```diff
@@ -195,6 +201,14 @@ <h1>
 			</div>
 		</div>
 	</article>
+		<div class="Previous-next">
```

Comment:
Use lowercase class name:

```suggestion
		<div class="previous-next">
```